Formula/wxmac.rb

```diff
@@ -64,7 +64,6 @@ class Wxmac < Formula
     end
   end
 
-  option :universal
```

Comment:
```
      # Otherwise, even in superenv, the internal libtiff can pick
      # up on a nonuniversal xz and fail
      # https://github.com/Homebrew/legacy-homebrew/issues/22732
      "--without-liblzma",
```
We probably don't need this argument anymore.